src/lib/BUILD.gn

```diff
@@ -53,10 +54,31 @@ static_library("lib") {
   }
 
   cflags = [ "-Wconversion" ]
+}
+
+static_library("lib") {
+  public_deps = [
+    ":base",
+    "${chip_root}/src/platform/logging:default",
+  ]
 
   output_name = "libCHIP"
 
   output_dir = "${root_out_dir}/lib"
 
   complete_static_lib = true
 }
+
+# This version of libCHIP is only meant for interal use by some test drivers
+static_library("lib_using_stdio") {
+  public_deps = [
+    ":base",
+    "${chip_root}/src/platform/logging:stdio",
+  ]
+
+  output_name = "libCHIP_using_stdio"
```

Comment:
Is `libCHIP_using_stdio` a good name? Any better ideas?

Reply:
Since most of lib names are in CamelCase in the project, how about `libCHIPWithConsoleLogging.a` ?

Reply:
To stay consistent I changed it to `libCHIPWithStdioLogging`